.travis.yml

```diff
@@ -22,22 +21,10 @@ branches:
 matrix:
   fast_finish: true
   include:
-    - php: 5.3
-      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
-    - php: 5.5
-      env: SYMFONY_VERSION='2.3.* symfony/expression-language:2.4.*'
-    - php: 5.5
-      env: SYMFONY_VERSION='2.3.* sensio/framework-extra-bundle:2.*'
-    - php: 5.5
-      env: SYMFONY_VERSION=2.4.*
-    - php: 5.5
-      env: SYMFONY_VERSION=2.6.*
-    - php: 5.5
-      env: SYMFONY_VERSION='2.7.*'
+    - php: 5.4
+      env: SYMFONY_VERSION=2.7.* COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
``COMPOSER_FLAGS="--prefer-lowest"`` must be done on a build which does not change the Symfony requirements at all, otherwise you are not actually testing the lowest bounds (as you modify them)